include/openPMD/backend/Attributable.hpp

```diff
@@ -97,16 +97,23 @@ class Attributable
      * @param   key     Key (i.e. name) to identify and store an Attributes value by.
      * @param   value   Value of Attribute stored with the provided key.
      * @return  true if key was already present, false otherwise
+     *
+     * @{
```

Comment:
Out of curiosity:
I am unfamiliar with this systax. Does this tell doxygen to generate only one doc for the both of these?

Reply:
Yes, exactly. It's doxyen group-ing for exactly that use case :)